styx-service-common/src/test/java/com/spotify/styx/storage/DatastoreStorageTest.java

```diff
@@ -261,17 +289,24 @@ public void tearDown() throws Exception {
   }
 
   @Test
-  public void shouldPersistWorkflows() throws Exception {
-    Workflow workflow = Workflow.create("test",
-                                        FULL_WORKFLOW_CONFIGURATION);
+  @Parameters(method = "configurations")
+  public void shouldPersistWorkflows(WorkflowConfiguration configuration) throws Exception {
+    Workflow workflow = Workflow.create("test", configuration);
     storage.store(workflow);
     Optional<Workflow> retrieved = storage.workflow(workflow.id());
 
     assertThat(retrieved, is(Optional.of(workflow)));
   }
 
+  private static WorkflowConfiguration[] configurations() {
+    return new WorkflowConfiguration[] {
+        FULL_WORKFLOW_CONFIGURATION, DOCKER_EXEC_WORKFLOW_CONFIGURATION, FLYTE_EXEC_WORKFLOW_CONFIGURATION
+    };
+  }
+
   @Test
-  public void shouldDeleteWorkflows() throws Exception {
+  @Parameters(method = "configurations")
+  public void shouldDeleteWorkflows(WorkflowConfiguration configuration) throws Exception {
```

Comment:
Not exactly sure what the `@Parameters` does, but the parameter to this function doesn't seem to be used inside it

Reply:
Done